src/app/organisms/room/RoomViewCmdBar.jsx

```diff
@@ -237,6 +238,7 @@ function RoomViewCmdBar({ roomId, roomTimeline, viewEvent }) {
       viewEvent.emit('cmd_fired');
     }
     if (myCmd.prefix === ':') {
+      if (!myCmd.result.mxc) addRecentEmoji(myCmd.result.unicode);
```

Comment:
same issue as above. we should not add custom emoji to recent for now as they are not used by wide variety of client. also we are updating `io.element.recent_emoji` which doesn't support custom emoji.